py54auth/registration_system/registration_service.py

```diff
@@ -0,0 +1,30 @@
+from instances.user import User
+from managers.user_manager import UserManager
+from settings import JSON_STORAGE
+from storage.json_storage import JsonStorage
+
+'''Регистрация пользователя'''
+
+
+class RegistrationSystem:
+    storage = JsonStorage(JSON_STORAGE)
+    user_manager = UserManager()
+    storage1 = storage.read()
```

Comment:
storage1 - not the best name for this var..
would be better to use just storage without storage1